packages/terra-site/src/examples/paginator/ProgressivePaginatorExample.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ProgressivePaginator from 'terra-paginator/src/ProgressivePaginator';
```

Comment:
We should import instead from the lib folder I believe.

Reply:
https://github.com/cerner/terra-core/pull/1445/commits/fd545ca9b18ee3883c109a58fe3be398ad95d18c